dds_gen/src/generator/mod.rs

```diff
@@ -1 +1,4 @@
+pub mod context;
 pub mod rust;
+
+pub use self::context::Context;
```

Comment:
This re-export won't make sense when the context is moved into the rust module so it should be removed.

In any case we are not fond of this because there are now two different ways of accessing Context and the re-export makes it confusing to know where it really is defined.